README.md

```diff
@@ -83,8 +83,8 @@ provide isolation. To replicate this environment,
 2.  install pkgr
 
 3.  open package in an R session and run `renv::init()`
```

Comment:
this is unrelated to this branch change, but I have been meaning to change this for awhile... this line should be `renv::init(bare = TRUE)`